OptimizelySDKUniversal/unexported_symbols.txt

```diff
@@ -1,73 +1,15 @@
 _BUCKETING_ID_TEMPLATE
-_OPDBBlockSQLiteCallBackFunction
-_OPDBExecuteBulkSQLCallback
 _HASH_SEED
-_OPJMModelErrorDomain
 _MAX_HASH_VALUE
 _MAX_TRAFFIC_VALUE
 _MurmurHash3_x64_128
 _MurmurHash3_x86_128
 _MurmurHash3_x86_32
-_OBJC_CLASS_$_OPDBDatabase
```

Comment:
if someone wanted to use FMDB and our universal framework. would this prevent them from using our modified FMDB?